docs/courseadministration/addcontent/learningunits/quiz_import.md

```diff
@@ -7,13 +7,14 @@ This is the preferred way for many, as it saves time and you can adapt and reuse
 ## 1. Make a copy of this template
 You will have to make a copy of our quiz template spreadsheet, that you can find below.
 
-![How our quiz template looks like](../../../img/courseadministration/quizmanagement/quiz_template.png)
+![How our quiz template looks like](../../../img/courseadministration/quizmanagement/quiz export.png)
 
-Find the non-to-be-edited quiz structure template [here](https://docs.google.com/spreadsheets/d/1_b8wO9_HlrrHQxagYan1iM6BBK0XtnD6vESxmhnML5s)
+Find the non-to-be-edited quiz structure template [here](https://docs.google.com/spreadsheets/d/1jqKQf4l5M-xPeDwyWl-M9XYzutcM-UsUqM7llMbaUpw/edit?usp=sharing)
 
 **Before proceeding, remember: Please, do not edit directly the template, but make a copy of it - see below how to do it.**
 
-![How to make a copy of a Google spreadsheet template](../../../img/courseadministration/quizmanagement/copy_quiz_template.png)
+![How to make a copy of a Google spreadsheet template](<img width="709" alt="image" src="https://user-images.githubusercontent.com/26278355/205922408-abf76038-2268-4729-bf51-97ee761fba29.png">)
```

Comment:
Hallo Steffi, hier muss der richtige Link zu dem Bild rein. So wie das im Original war, nur eben mit dem neuen Bild. Das Bild muss vorher in den entsprechenden Ordner hochgeladen werden. So wie es hier ist funktioniert das nur in GitHub aber nachher nicht in Read the Docs